redaxo/src/core/lib/login/login.php

```diff
@@ -333,12 +334,14 @@ public function checkLogin()
         } else {
             $this->message = rex_i18n::msg('login_logged_out');
 
+            rex_user_session::clearCurrentSession();
             rex_csrf_token::removeAll();
         }
 
         if ($ok) {
             // wenn alles ok dann REX[UID][system_id] schreiben
             $this->setSessionVar(self::SESSION_LAST_ACTIVITY, time());
+            rex_user_session::updateLastActivity();
```

Comment:
Ich fürchte, dass wir das eher in `rex_backend_login` integrieren müssen, statt hier in der Basisklasse.
Also wer die Basis-Klasse für irgendeinen eigenen Loginbereich nutzt, bei dem sollten die Sessions nicht mit in unsere Tabelle für die Backend-Sessions geschrieben werden.